### PR TITLE
Specify the color space (sRGB) in all numpy.org CSS files

### PR DESCRIPTION
**Pull request by aerikpawson**
Fixes #numpygh-397.

### CODE REVIEW

**Comment by InessaPawson** [comment]
@aerikpawson LGTM (Looks good to me.) One suggestion: next time, before pushing, squash all the commits you'd like to include in a pull request into one. You’ll have to use the terminal for that.

**Review by InessaPawson — APPROVED** [review]

File: static/css/styles.css
Comment (joelachance) [inline static/css/styles.css:350]: we should remove the space here

File: static/css/styles.css
Comment (joelachance) [inline static/css/styles.css:8]: I personally think it's really nice to add a description here for each color-- I don't have any idea what each of these are, and it will be easier to use them in the future if we know which one is the blue one, for example.

**Review by joelachance — APPROVED** [review]
Looks good, a couple of small things I would add, but otherwise everything looks good to me!

File: static/css/styles.css
Comment (aerikpawson) [inline static/css/styles.css:350]:
```suggestion
        color: rgb(0, 0, 0);
```

File: static/css/styles.css
Comment (InessaPawson) [reply static/css/styles.css:8]: @joelachance It’s a fair point. The issue with the current color names is that some of them have been assigned to a different RGB since we selected the color palette for numpy.org. (For more context, refer to https://github.com/numpy/numpy.org/pull/405#issuecomment-798808702.) This is bound to cause confusion.

What do you think of naming the palette as follows:
rgb(255, 197, 83) NumPy Yellow (or NumPy Mustard)
rgb(77, 171, 207) NumPy Ndarray Blue
rgb(77, 119, 207) NumPy Deep Blue
rgb(255, 255, 255) White
rgb(238, 238, 238) NumPy Cloud Gray 
rgb(108, 122, 137) NumPy Slate Gray
rgb(1, 50, 67) NumPy Warm Black

@joelachance and everyone else, feel free to suggest your names for the color palette.

Comment (joelachance) [reply static/css/styles.css:8]: I like them! I think any color description that's close to the color is 'good enough'. Developers will easily know what color they're using with any description.

Comment (InessaPawson) [reply static/css/styles.css:8]: @joelachance Great! 
@aerikpawson Please incorporate the suggested above into your PR.

**Review by joelachance — APPROVED** [review]
LGTM!

**Review by InessaPawson — APPROVED** [review]
Looks great, Aerik! Keep it up!

**Comment by InessaPawson** [comment]
> LGTM!

@joelachance I don't have the push access to this repo. Please merge this PR when you get a chance.

**Pull request merged**
